tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/conversion/NativeConversionNegativeTest.java

```diff
@@ -87,14 +87,6 @@ public void testTupleConversionFail() {
                 + "T2]'");
     }
 
-    @Test(description = "Test converting an unsupported array to json")
```

Comment:
Instead of removing this and the tests in https://github.com/ballerina-platform/ballerina-lang/pull/20531/files#diff-17ddc3514c16ada932b765cb849594b1L228, shall we change the tests to use some other non-JSON compatible field?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/20531/commits/091f20ee401e21f690630dd8cf20c16f1e9a1b11